src/screens/DiscoveryScreenV2/fragment/NewsFragment.js

```diff
@@ -139,11 +139,11 @@ const NewsFragment = ({
 const styles = StyleSheet.create({
   fragmentContainer: {
     flex: 1,
-    backgroundColor: COLORS.concrete
+    backgroundColor: COLORS.gray100
   },
   noDataFoundContainer: {
     flex: 1,
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.white` to `COLORS.almostBlack` for the `backgroundColor` of `noDataFoundContainer` also seems fine if it aligns with the new color scheme. Again, ensure that this color change doesn't affect readability or accessibility (contrast) in this component.


<!-- This is an auto-generated comment by OSS CodeRabbit -->